js/constants/appConfig.js

```diff
@@ -187,7 +188,7 @@ module.exports = {
     // payments
     'payments.allow-media-publishers': true,
     'payments.allow-non-verified-publishers': true,
-    'payments.contribution-amount': 10, // BAT
+    'payments.contribution-amount': null, // BAT
```

Comment:
The actual value (for new installs only) is set to null to avoid "locking in" a value